src/Resolvers/WorkloadManifestValidator/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace WorkloadManifestValidator
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
This looks like just a stub project.  Do we need it?

Reply:
Nope, I didn't get round to implementing this. It should be punted to a later PR.